docs/configuration.md

```diff
@@ -0,0 +1,96 @@
+<!--
+    Copyright 2019 VMware, Inc.
+    SPDX-License-Identifier: BSD-2-Clause
+-->
+
+# Precaution configuration
+
+Precaution is using a variety of tools under the hood without exposing
+implementation details and complexity of the underlying tools to the user.
+
+Concretely, this means that Precaution ignores any tool specific configuration
+files which may exist in the project's repository and performs scans based only
+on Precaution specific configuration.
+
+Precaution provides you with the option to use optional `.precaution.json` file.
```

Comment:
Why JSON for config and not YAML?  YAML is much more human friendly readable/writable config nowadays.

Reply:
I agreed with you in the past pull request:
https://github.com/vmware/precaution/pull/213#pullrequestreview-237625980